main.js

```diff
@@ -1,30 +1,158 @@
 const tabItems = document.querySelectorAll(".tab-item");
 const tabContentItems = document.querySelectorAll(".tab-content-item");
+const inProgressItems = document.querySelectorAll(".color-c");
+// functia removeShow face sa nu apara continutul tututor taburilor
 removeShow();
+// selecteaza pagina (tabul) care sa se afiseze initial si butonul acesteia activat
 document.querySelector("#tasks-content").classList.add("show");
 document.querySelector("#tasks").classList.add("active");
 
 function selectItem(e) {
-removeShow();
-console.log(this.id);
-const tabItem = document.querySelector(`#${this.id}`);
-const tabContentItem = document.querySelector(`#${this.id}-content`);
-console.log(tabContentItem);
-tabContentItem.classList.add("show");
-tabItem.classList.add("active");
+  removeShow();
+  console.log(this.id);
+  const tabItem = document.querySelector(`#${this.id}`);
+  const tabContentItem = document.querySelector(`#${this.id}-content`);
+  console.log(tabContentItem);
+  tabContentItem.classList.add("show");
+  tabItem.classList.add("active");
 }
 
 function removeShow() {
   tabContentItems.forEach((item) => item.classList.remove("show"));
   tabItems.forEach((item) => item.classList.remove("active"));
 }
 
-tabItems.forEach(item => {
-	item.addEventListener('click', selectItem);
+tabItems.forEach((item) => {
+  item.addEventListener("click", selectItem);
 });
 
-$('#myList a').on('click', function (e) {
-    e.preventDefault()
-    $('#myList a[href="#profile"]').tab('show')
-    $(this).tab('show')
-  })
\ No newline at end of file
+// butonul close alaturat fiecarui nod existent
+var myNodelist = document.getElementsByTagName("LI");
+var i;
```

Comment:
inlocuieste in toata aplicatia var-urile cu let sau const, in functie de ce ai nevoie.